howso/engine/trainee.py

```diff
@@ -453,6 +453,7 @@ def copy(
         name: t.Optional[str] = None,
         *,
         library_type: t.Optional[LibraryType] = None,
+        persistence: t.Optional[Persistence] = None,
```

Comment:
Since this parameter was rolled back in the client, need to also remove this change from here